.github/workflows/dbdocs.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow generates database documentation and ERD files.
+#
+name: Generate dbdocs
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/dbdocs.yml
+      - db/migrations/bcda
+      - db/migrations/bcda_queue
+
+env:
+  VAULT_PW: ${{ secrets.VAULT_PW }}
+
+jobs:
+  generate-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Decrypt secrets
+        run: |
+          echo $VAULT_PW > .vault_password
+          bash ops/secrets --decrypt
+          mv -fv shared_files/encrypted/* shared_files/decrypted/
+
+      - name: Start DB
+        run: make start-db
+
+      - name: Generate docs
+        run: docker run --rm -v $PWD:/work -w /work --network bcda-app-net ghcr.io/k1low/tbls doc --rm-dist "postgres://postgres:toor@db:5432/bcda?sslmode=disable" dbdocs/bcda
```

Comment:
Should/can we try to get the user/pw (other vars?) from env vars?

Reply:
These are hardcoded in other make commands, so I left that as is. I ended up moving this into `make dbdocs` though so if those values change, they can be updated in one file instead of multiple different places.